pyproject.toml

```diff
@@ -18,17 +18,26 @@ packages = [{ include = "modmail" }]
 [tool.poetry.dependencies]
 python = "^3.8"
 aiohttp = { extras = ["speedups"], version = "^3.7.4" }
+arrow = "^1.1.1"
 colorama = "^0.4.3"
 coloredlogs = "^15.0"
 "discord.py" = {git = "https://github.com/Rapptz/discord.py.git", rev = "master"}
+environs = {version = "~=9.3.3", optional = true}
 pydantic = "^1.8.2"
 python-dotenv = "~=0.17.1"
```

Comment:
Why are we using two env loaders here? Choose one, I had say to go with `environs` until we get configurave in a stable state.

Reply:
I do not want to touch the config file until we have a different thing for it, but if you insist...

Reply:
I think we ought leave this for cleanup until we make a configurave integration pr

Reply:
Perfect